### PR TITLE
Refactor:Cleanup Naming and Optimize Email Digest Code

### PR DESCRIPTION
**Pull request by mstruve**
## What type of PR is this? (check all applicable)
- [x] Refactor
- [x] Optimization

## Description
- Move Logic intensive Email Digest code into Services, we are deprecating labor
- Optimization: Grab user_ids in batches
- Use plural Emails instead of Email to match with convention

alt_text

### CODE REVIEW

File: app/workers/emails/enqueue_digest_worker.rb [hunk 1]
```diff
@@ -1,4 +1,4 @@
-module Email
+module Emails
```
Comment (rhymes) [inline app/workers/emails/enqueue_digest_worker.rb:1]: IIRC when the name of a worker is changed we should add an alias for a while, right?

File: app/services/email_digest.rb [hunk 2]
```diff
@@ -8,8 +8,10 @@ def initialize(users = [])
   end
 
   def send_periodic_digest_email
-    @users.ids.each do |user_id|
-      Email::SendUserDigestWorker.perform_async(user_id)
+    @users.select(:id).in_batches do |batch|
```
Comment (rhymes) [inline app/services/email_digest.rb:11]: :-)

**Review by rhymes — COMMENTED** [review]
neat

File: app/workers/emails/enqueue_digest_worker.rb (hunk 1 above)
Comment (mstruve) [reply app/workers/emails/enqueue_digest_worker.rb:1]: Nah, this will be fine and I actually want this right now bc I need to kill the job running.

**Comment by mstruve** [comment]
Admin merged to kill the current job running which is overwhelming our database

**Pull request merged**